OpenRA.Game/Graphics/CursorProvider.cs

```diff
@@ -36,9 +36,24 @@ public CursorProvider(ModData modData)
 					out shadowIndex[shadowIndex.Length - 1]);
 			}
 
+			var paletteLoaders = new Dictionary<string, string>();
+			if (nodesDict.ContainsKey("PaletteLoaders"))
```

Comment:
Please use `nodesDict.TryGetValue()` instead of checking if it exists and then querying it.